### PR TITLE
Added release notes for 1.5

### PR DESCRIPTION
**Pull request by teresaromero**
Signed-off-by: Teresa Romero <teresa@okteto.com>

### CODE REVIEW

File: src/content/self-hosted/install/releases.mdx [hunk 1]
```diff
@@ -5,6 +5,48 @@ sidebar_label: Release Notes
...
+## 1.5.0
+14 February, 2023
+
+### Breaking Change
+
+- Create dedicated registry serviceAccounts by default
+
+### Features
+
+- Added affinity/anti option to PODs with label dev.okteto.com/affinity
```
Comment (pchico83) [inline src/content/self-hosted/install/releases.mdx:17]: Can we add a link to the docs?

File: src/content/self-hosted/install/releases.mdx [hunk 2]
```diff
@@ -5,6 +5,48 @@ sidebar_label: Release Notes
...
+- Add external resource icons
```
Comment (pchico83) [inline src/content/self-hosted/install/releases.mdx:18]: Can we add a link to the docs?

File: src/content/self-hosted/install/releases.mdx [hunk 3]
```diff
@@ -5,6 +5,48 @@ sidebar_label: Release Notes
...
+- Build credential helpers
```
Comment (pchico83) [inline src/content/self-hosted/install/releases.mdx:19]: Can we add a link to the docs?

File: src/content/self-hosted/install/releases.mdx [hunk 4]
```diff
@@ -5,6 +5,48 @@ sidebar_label: Release Notes
...
+- Remove load time on GitHub redeploy dialog
+- Support common prefix namespaces
```
Comment (pchico83) [inline src/content/self-hosted/install/releases.mdx:21]: This is in `unsupported`, we shouldn't mention it here or document it

File: src/content/self-hosted/install/releases.mdx [hunk 5]
```diff
@@ -5,6 +5,48 @@ sidebar_label: Release Notes
...
 ---
 
+## 1.5.0
+14 February, 2023
+
+### Breaking Change
+
+- Create dedicated registry serviceAccounts by default
+
+### Features
```
Comment (pchico83) [inline src/content/self-hosted/install/releases.mdx:15]: The is also support for network policies templates

**Review by pchico83 — CHANGES_REQUESTED** [review]

File: src/content/self-hosted/install/releases.mdx (hunk 5 above)
Comment (teresaromero) [reply src/content/self-hosted/install/releases.mdx:15]: I dont understand this, can you elaborate more? You mean a new feature that is missing or is it related with the dedicated registry sa?

File: src/content/self-hosted/install/releases.mdx (hunk 2 above)
Comment (teresaromero) [reply src/content/self-hosted/install/releases.mdx:18]: This needs to be merged to release-1.5 https://github.com/okteto/docs/pull/300 ?

File: src/content/self-hosted/install/releases.mdx [hunk 6]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+
+- Added antiAffinity (configuration)[/docs/self-hosted/administration/configuration/#userpodaffinity] for PODs with label `dev.okteto.com/affinity`
+- Add external resource icons
+- Build credential helpers
+- Remove load time on GitHub redeploy dialog
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:20]: what does it mean to remove the load time?

File: src/content/self-hosted/install/releases.mdx [hunk 7]
```diff
@@ -5,6 +5,48 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.5.0
+14 February, 2023
+
+### Breaking Change
+
+- Create dedicated registry serviceAccounts by default
```
[the remark this answers is not in the document]
Comment (rberrelleza) [reply src/content/self-hosted/install/releases.mdx:13]: It's very confusing to talk about `default` account but then have to put the value on `.Values.registry`. Is this the account for the registry, or is this used for anything else?

[the remark this answers is not in the document]
Comment (rberrelleza) [reply src/content/self-hosted/install/releases.mdx:13]: What is the default value of `Values.registry.serviceAccountName` ?

File: src/content/self-hosted/install/releases.mdx (hunk 3 above)
Comment (rberrelleza) [reply src/content/self-hosted/install/releases.mdx:19]: what does "buil credential helpers" means? What scenarios does this enable?

File: src/content/self-hosted/install/releases.mdx [hunk 8]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+
+### Features
+
+- Added antiAffinity (configuration)[/docs/self-hosted/administration/configuration/#userpodaffinity] for PODs with label `dev.okteto.com/affinity`
+- Add external resource icons
+- Build credential helpers
+- Remove load time on GitHub redeploy dialog
+
+### Bug Fix
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:22]:
```suggestion
### Bugfixes
```

File: src/content/self-hosted/install/releases.mdx [hunk 9]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.5.0
+14 February, 2023
+
+### Breaking Change
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:11]:
```suggestion
### Breaking Changes
```

File: src/content/self-hosted/install/releases.mdx [hunk 10]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Filter repositories results by search term
+- Fix pointer cursor in staged log lines
+- Autopreviews are now being deleted when PR is merged or closed
+- Align endpoints and repos on `ResourceDetails`
+- Remove leftover configuration from the cert-manager days
+- Fix logged data on SSE connection failure
+- Fix typo in OPENID_RESOURCE envvar for ADFS
+- Check for nil pointer when accessing `Spec.IngressClassName`
+- Get claims from access token when `ADFS` resource is given
+- Fix 1.24 in EKS
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:44]: Does thi mean that we fully support Kubernetes 1.24? If so, this should be a feature.

File: src/content/self-hosted/install/releases.mdx [hunk 11]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+## 1.5.0
+14 February, 2023
+
+### Breaking Change
+
+- Create dedicated registry serviceAccounts by default
+
+### Features
+
+- Added antiAffinity (configuration)[/docs/self-hosted/administration/configuration/#userpodaffinity] for PODs with label `dev.okteto.com/affinity`
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:17]: - What is the user benefit of this feature? What escenarios does this enable?

File: src/content/self-hosted/install/releases.mdx [hunk 12]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Add external resource icons
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:18]: "You can now define an icon for every endpoint in your external resources list"

File: src/content/self-hosted/install/releases.mdx [hunk 13]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Build credential helpers
+- Remove load time on GitHub redeploy dialog
+
+### Bug Fix
+
+- When injecting host alises to go through internal network to buildkit, registry and api, don't duplicate entries for same IP. Instead, pass the list of hostnames for the IP
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:24]: What is this fixing?

File: src/content/self-hosted/install/releases.mdx [hunk 14]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Delete button is now disabled in the personal namespace
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:25]: Can we move this to the improvements section? And what is the reason/user benefit behind this?

File: src/content/self-hosted/install/releases.mdx [hunk 15]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Restore k3s support
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:26]: - Since we never officially supported this, I think we should move this to features as "Support for k3s" . Do we support an specific version? @pchico83 are we ready to officially support this?

File: src/content/self-hosted/install/releases.mdx [hunk 16]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Fix faulty error case on deployment with external resource
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:27]: What user-facing escenario is this fixing?

File: src/content/self-hosted/install/releases.mdx [hunk 17]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Fix faulty progressing state on deployment with external resource
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:28]: What user-facing escenario is this fixing?

File: src/content/self-hosted/install/releases.mdx [hunk 18]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Disable CTA while destroying all
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:29]: What do we mean by CTA here? The banner?

File: src/content/self-hosted/install/releases.mdx [hunk 19]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Catch all non `/error/{code}` requests
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:30]: What user-facing escenario is this fixing?

File: src/content/self-hosted/install/releases.mdx [hunk 20]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Make env var values editable on deploy/redeploy
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:31]: I think this is better in the "improvements" section.

File: src/content/self-hosted/install/releases.mdx [hunk 21]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Keep ingress class on wake if ingress class is different than "sleeping"
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:32]: I don't understand this one. What does it mean for the ingress class to be sleeping?

File: src/content/self-hosted/install/releases.mdx [hunk 22]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Fix service unavailable when destroy all
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:33]: What user-facing escenario is this fixing?

File: src/content/self-hosted/install/releases.mdx [hunk 23]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Fix divert snippet configuration
+- Filter repositories results by search term
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:35]: Can we move this to improvements?

File: src/content/self-hosted/install/releases.mdx [hunk 24]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Delete button is now disabled in the personal namespace
+- Restore k3s support
+- Fix faulty error case on deployment with external resource
+- Fix faulty progressing state on deployment with external resource
+- Disable CTA while destroying all
+- Catch all non `/error/{code}` requests
+- Make env var values editable on deploy/redeploy
+- Keep ingress class on wake if ingress class is different than "sleeping"
+- Fix service unavailable when destroy all
+- Fix divert snippet configuration
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:34]: What user-facing escenario is this fixing?

File: src/content/self-hosted/install/releases.mdx [hunk 25]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Filter repositories results by search term
+- Fix pointer cursor in staged log lines
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:36]: What does this fix?

File: src/content/self-hosted/install/releases.mdx [hunk 26]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Autopreviews are now being deleted when PR is merged or closed
+- Align endpoints and repos on `ResourceDetails`
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:38]: What is ResourceDetails? Is this a part of the UI?

File: src/content/self-hosted/install/releases.mdx [hunk 27]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Remove leftover configuration from the cert-manager days
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:39]: "Removed unused configuration values related to cert-manager in the Helm chart"

File: src/content/self-hosted/install/releases.mdx [hunk 28]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Fix logged data on SSE connection failure
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:40]: What do we mean by logged data?

File: src/content/self-hosted/install/releases.mdx [hunk 29]
```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
...
+- Fix typo in OPENID_RESOURCE envvar for ADFS
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:41]: "Fixed issue in the OpenID configuration that prevented using    ADFS as the identity provider"

File: src/content/self-hosted/install/releases.mdx (hunk 6 above)
Comment (teresaromero) [reply src/content/self-hosted/install/releases.mdx:23]: @bgonp this https://github.com/okteto/app/pull/5607 is tagged as feature, can you elaborate a bit to know the context?

File: src/content/self-hosted/install/releases.mdx (hunk 15 above)
Comment (teresaromero) [reply src/content/self-hosted/install/releases.mdx:34]: @pchico83 @jpf-okteto @rberrelleza shoudl this be removed from release notes then?

File: src/content/self-hosted/install/releases.mdx (hunk 16 above)
Comment (teresaromero) [reply src/content/self-hosted/install/releases.mdx:35]: @viddo https://github.com/okteto/app/pull/5447 This is reported as bug-fix, can we elaborate on this for context?

File: src/content/self-hosted/install/releases.mdx (hunk 17 above)
Comment (teresaromero) [reply src/content/self-hosted/install/releases.mdx:36]: @viddo https://github.com/okteto/app/pull/5449 same on this one

File: src/content/self-hosted/install/releases.mdx (hunk 18 above)
Comment (teresaromero) [reply src/content/self-hosted/install/releases.mdx:37]: @tavferreira https://github.com/okteto/app/pull/5453 can you give more contexT?

File: src/content/self-hosted/install/releases.mdx (hunk 25 above)
Comment (teresaromero) [reply src/content/self-hosted/install/releases.mdx:42]: @rlamana https://github.com/okteto/app/pull/5537 can you give more contexT?

File: src/content/self-hosted/install/releases.mdx (hunk 26 above)
Comment (teresaromero) [reply src/content/self-hosted/install/releases.mdx:44]: @tavferreira https://github.com/okteto/app/pull/5538 can you give more context on this one?

File: src/content/self-hosted/install/releases.mdx (hunk 28 above)
Comment (teresaromero) [reply src/content/self-hosted/install/releases.mdx:46]: @viddo https://github.com/okteto/app/pull/5609 same with this one

File: src/content/self-hosted/install/releases.mdx (hunk 6 above)
Comment (bgonp) [reply src/content/self-hosted/install/releases.mdx:23]: When redeploying a resource, we've changed the way we are checking if the user has enough permissions to handle a repository and the way we are fetching the repository data so the user experience has been improved by needing less time to trigger the same action.

I guessed that `release/new-feature` was the right label to use, but I might be wrong...

File: src/content/self-hosted/install/releases.mdx (hunk 18 above)
Comment (tavferreira) [reply src/content/self-hosted/install/releases.mdx:36]: Sorry, I should have been more descriptive here. I'll keep it in mind in the future.

It refers to the `Launch dev environment` CTA when a space is empty. 
In some cases, the button was enabled even when the destroying action was still taking action.

File: src/content/self-hosted/install/releases.mdx (hunk 25 above)
Comment (tavferreira) [reply src/content/self-hosted/install/releases.mdx:41]: Jumping in.
Resource log lines were inheriting a `cursor` property of type `pointer` from the parent container, which was giving the idea that log lines were actionable, when they weren't.
This PR reverts to use default cursor.

File: src/content/self-hosted/install/releases.mdx (hunk 6 above)
Comment (rlamana) [reply src/content/self-hosted/install/releases.mdx:23]: Maybe we can rephrase it to `Reduced load time when opening Github redeploy dialog` or something like that.

File: src/content/self-hosted/install/releases.mdx (hunk 25 above)
Comment (rlamana) [reply src/content/self-hosted/install/releases.mdx]: Maybe it should say `Fixed wrong cursor when hovering logs lines, they can now be selected as normal text`

**Review by pchico83 — APPROVED** [review]

**Review by rberrelleza — APPROVED** [review]
